src/order/zorn.lean

```diff
@@ -240,13 +242,19 @@ exists_maximal_of_chains_bounded
 
 end chain
 
+--This lemma isn't under section `chain` because `parameters` messes up with it. Feel free to fix it
+theorem chain.symm {α : Type u} {s : set α} {q : α → α → Prop} (h : chain q s) :
```

Comment:
```suggestion
/-- This can be used to turn `zorn.chain (≥)` into `zorn.chain (≤)` and vice-versa. -/
theorem chain.symm {α : Type u} {s : set α} {q : α → α → Prop} (h : chain q s) :
```